apps/src/templates/tables/QuickActionsCell.jsx

```diff
@@ -81,17 +118,22 @@ class QuickActionsCell extends Component {
   };
 
   render() {
+    const {type} = this.props;
     const targetPoint = {top: this.state.menuTop, left: this.state.menuLeft};
 
+    const icons = {
+      header: 'cog',
+      body: 'chevron-down'
+    };
+
     return (
       <span ref={span => this.icon = span}>
-        <a
-          icon="chevron-down"
-          style={this.state.open ? {...styles.actionButton, ...styles.hoverFocus} : styles.actionButton}
+        <FontAwesome
+          icon={icons[type]}
+          style={this.state.open ? styles.hoverFocus[type] : styles.actionButton[type]}
           onClick={this.state.canOpen ? this.open : undefined}
-        >
-          <i className="fa fa-chevron-down ui-test-section-dropdown"></i>
-        </a>
+          className="ui-test-section-dropdown"
```

Comment:
- [ ] double check that this didn't break the other uses of this dropdown 
- [ ] ensure that the UI tests that depend on the section dropdown still pass